corpora-config.json

```diff
@@ -0,0 +1,82 @@
+{
+  "corpora": [
+    { "name": "Accessibility",
+      "baseurl": "https://pages.18f.gov",
+      "indexPath": "/accessibility/search-index.json",
```

Comment:
Do the search indexes include the path prefix (`/accessibility/`) on Pages sites?